src/agents/state.py

```
@@ -1 +1,82 @@
+    feature_results: Annotated[List[Dict[str, Any]], operator.add]
+    solve_result: Optional[SolveResult] = None
+    explain_result: Optional[ExplainResult] = None
+    graph_result: Optional[GraphResult] = None
+    variant_result: Optional[VariantResult] = None
+    solution_result: Optional[SolutionResult] = None
+
+    # Review Layer
+    review_state: ReviewState
+
+    # Tool/최종 (기본값 추가)
+    tool_outputs: Dict[str, Any] = {}
```

Ruff (0.14.11)

Mutable class attributes should be annotated with `typing.ClassVar`

(RUF012)